src/pages/case-studies/ibm-commerce-platform.mdx

```diff
@@ -0,0 +1,333 @@
+---
+title: IBM Commerce Platform
+description:
+  This case study examines the adoption of Carbon for the design and development
+  of the new self-service purchase experience on IBM.com.
+---
+
+<PageDescription>
+
+Learn about how the IBM.com team adopted Carbon for the design and development
+of the new self-service purchase experience on IBM.com.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Summary</AnchorLink>
+<AnchorLink>The challenge</AnchorLink>
+<AnchorLink>Proposed solution</AnchorLink>
+<AnchorLink>Our approach</AnchorLink>
+<AnchorLink>Benefits</AnchorLink>
+<AnchorLink>Positive outcomes</AnchorLink>
+<AnchorLink>Next steps</AnchorLink>
+<AnchorLink>Related resources</AnchorLink>
+
+</AnchorLinks>
+
+## Summary
+
+The [IBM.com](http://ibm.com/) Commerce Platform team spent 3 three months
+transitioning their checkout flow to use Carbon to improve their conversion rate
+and overall product experience. They took an iterative approach,
+starting first with small, isolated features to get comfortable designing and
+developing with Carbon. Next they replaced existing elements with Carbon
+components. Once all the components had been transitioned, the team focused on
+updating page structure and layout.
+
+Since the Carbon refresh, the team’s delivery cycle has sped up and they spend
```

Comment:
I think we should change the term "Carbon refresh" to "Carbon implementation" (it's used in several places in the text). "Refresh" implies (to me, at least) just a visual tweak.

Reply:
Fixed!